cmd/buildah/main.go

```diff
@@ -90,7 +93,11 @@ func main() {
 	}
 	err := app.Run(os.Args)
 	if err != nil {
-		logrus.Errorf("%v", err)
-		os.Exit(1)
+		if debug {
+			logrus.Errorf(err.Error())
+		} else {
+			fmt.Fprintln(os.Stderr, err.Error())
+		}
+		cli.OsExiter(1)
```

Comment:
I might be losing my marbles, but wasn't this change already made?

Reply:
It was made in kpod but not in buildah.